### DISCUSSION TITLE
How to setup the notes_path?

### DISCUSSION THREAD

**Discussion by bohmbox** [Furkanzmc/zettelkasten.nvim#27] [General]
I have a question, I installed the plugin for the zettelkasten.nvim project using an ubuntu machine, running neovim-lua (lunarvim). The problem is I can't figure out how to set the **notes_path** to an address I know.  
Where or how do I do that?. The guides and documentation are too scant regarding this.  
thanks

**Comment by Furkanzmc** [comment]
Hello Rodolfo.

You can call the set up function at any point you prefer. I usually have it set up in my machine local vimrc file since the location might change for me. For example, you could do something like this in `~/.vimrc`

```vim
lua require("zettelkasten").setup({notes_path="~/path/to/notes"})
```

**Reply by bohmbox** [reply]
hey, thanks for the answer. That helped!
